content/docs/en/elements/components/list-view.md

```diff
@@ -1,5 +1,11 @@
 ---
 title: ListView
+apiRef: https://docs.nativescript.org/api-reference/classes/_ui_list_view_.listview
+
+---
+
+The ListView shows items in a vertically scrolling list. You can set an itemTemplate to specify how each item in the list should be displayed.
```

Comment:
This is somewhat true, but will need a bit of work because of how we handle list-views with the `v-template` component. 

A good example of the syntax can be found here: https://github.com/nativescript-vue/nativescript-vue/blob/master/samples/app/app-with-v-template.js#L35-L51

Reply:
Didn't know about that - I'll read through it later.